### ISSUE TITLE
Null Safety version

### ISSUE THREAD

**Issue by victormanuelfrancodev** [apeiro-tech/conekta_flutter#3]
Actually, I'm Working with your package, but I need the version for null safety 
```

code's output:
↳
    Error: Cannot run with sound null safety, because the following dependencies
    don't support null safety:

     - package:conekta_flutter
```

#2 Please

**Comment by aosorio-avilez** [comment]
I will close the issue, this is fixed on version 0.0.4-nullsafety, thanks for your contribution 👍🏿

**Comment by InvenceSaltillo** [comment]
I have version 0.0.4-nullsafety but have same error

code's output:
↳
    Error: Cannot run with sound null safety, because the following dependencies
    don't support null safety:

     - package:conekta_flutter
